spec/lib/search_spec.rb

```diff
@@ -3135,4 +3135,30 @@ def indexed_post(*args)
       expect(results.posts).to contain_exactly(regular_post)
     end
   end
+
+  it "can order by read" do
```

Comment:
```suggestion
  it "orders posts by the timestamp of the user's last visit to each post's topic" do
```

Suggestion to make it clearer since just saying `read` leaves room for interpretation.